### ISSUE TITLE
Support Android

### ISSUE THREAD

**Issue by cryptono** [pfn/keepasshttp#319]
Would love to see this working with https://addons.mozilla.org/de/android/addon/keepasshelper/?src=search on Android or anything else.

**Comment by pfn** [comment]
https://play.google.com/store/apps/details?id=com.hanhuy.android.keepshare.lite
https://play.google.com/store/apps/details?id=com.hanhuy.android.keepshare
https://github.com/pfn/keepshare

doesn't work with firefox, however, they did not implement real accessibility support.
